templates/stats/issues.tpl

```diff
@@ -0,0 +1,199 @@
+{**
+ * templates/stats/issues.tpl
+ *
+ * Copyright (c) 2022 Simon Fraser University
+ * Copyright (c) 2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * The issues statistics page.
+ *
+ *}
+{extends file="layouts/backend.tpl"}
+
+{block name="page"}
+
+	<div class="pkpStats">
+		<pkp-header>
+			<h1>{translate key="issue.issues"}</h1>
+			<spinner v-if="isLoadingTimeline"></spinner>
+			<template slot="actions">
+				<date-range
+					unique-id="issue-stats-date-range"
+					:date-start="dateStart"
+					:date-end="dateEnd"
+					:date-end-max="dateEndMax"
+					:options="dateRangeOptions"
+					date-range-label="{translate key="stats.dateRange"}"
+					date-format-instructions-label="{translate key="stats.dateRange.instructions"}"
+					change-date-range-label="{translate key="stats.dateRange.change"}"
+					since-date-label="{translate key="stats.dateRange.sinceDate"}"
+					until-date-label="{translate key="stats.dateRange.untilDate"}"
+					all-dates-label="{translate key="stats.dateRange.allDates"}"
+					custom-range-label="{translate key="stats.dateRange.customRange"}"
+					from-date-label="{translate key="stats.dateRange.from"}"
+					to-date-label="{translate key="stats.dateRange.to"}"
+					apply-label="{translate key="stats.dateRange.apply"}"
+					invalid-date-label="{translate key="stats.dateRange.invalidDate"}"
+					date-does-not-exist-label="{translate key="stats.dateRange.dateDoesNotExist"}"
+					invalid-date-range-label="{translate key="stats.dateRange.invalidDateRange"}"
+					invalid-end-date-max-label="{translate key="stats.dateRange.invalidEndDateMax"}"
+					invalid-start-date-min-label="{translate key="stats.dateRange.invalidStartDateMin"}"
+					@set-range="setDateRange"
+				></date-range>
+			</template>
+		</pkp-header>
+		<div class="pkpStats__container -pkpClearfix">
+			<div class="pkpStats__content">
+				<div v-if="chartData" class="pkpStats__graph">
+					<div class="pkpStats__graphHeader">
+						<h2 class="pkpStats__graphTitle -screenReader" id="issue-stats-graph-title">
+							{translate key="submission.views"}
+						</h2>
+						<div class="pkpStats__graphSelectors">
+							<div class="pkpStats__graphSelector pkpStats__graphSelector--timelineType">
+								<pkp-button
+									:aria-pressed="timelineType === 'toc'"
+									aria-describedby="issue-stats-graph-title"
+									@click="setTimelineType('toc')"
+								>
+									{translate key="issue.toc"}
+								</pkp-button>
+								<pkp-button
+									:aria-pressed="timelineType === 'files'"
+									aria-describedby="issue-stats-graph-title"
+									@click="setTimelineType('files')"
+								>
+									{translate key="editor.issues.galleys"}
+								</pkp-button>
```

Comment:
When selecting the galleys button in the chart, it still shows me stats for the table of contents views.

Reply:
Good catch!!! It is a bug in the API -- I forgot to consider the type parameter...